resources/Resource.go

```diff
@@ -282,7 +282,8 @@ func stringToBool(s string) bool {
 }
 
 func parseIntsFromString(s string) ([]int, error) {
-	strings := strings.Split(s, ",")
+	str := strings.TrimSpace(s)
+	strings := strings.Split(str, ",")
```

Comment:
Other way around :slightly_smiling_face:
This will help only if the string looks like `"     aaa,bbb,ccc     "`. But what if the string looks like `"  aaa,    bbb,    ccc    ,dddd    "`? You have to first split it and then trim each part.